addons/stock/populate/product.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models
+
+PRODUCT_TYPES = ["consu", "service", "product"]
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _get_types(self):
```

Comment:
Inherit example, nice! but actually, I would prefer a call to super. Good to know: the sum of weight doesn't need to be 1 or 100

```
 def _populate_get_types(self):
        return ['consu', 'service'], [2, 1]

def _populate_get_types(self):
      types, weights = super()._get_types()
      return types+['product'], weights+[2]
```
Also, prefix with populate. For now, populate methods are available everywhere, let's avoid method names clash.